src/js/modules/edit.js

```diff
@@ -1631,9 +1632,7 @@ Edit.prototype.editors = {
 					item.element = el;
 
 					if(intialLoad && item.value == initialValue){
-						input.value = item.title;
-						item.element.classList.add("active");
-						current = true;
+						autoSelectIndex = idx;
 					}
```

Comment:
The case for initialValue did not actually work with keyUp/keyDown prior to the addition of editorParams.autoSelectFirstItem but I stumbled on that bug while testing my changes.

The issue was that if the initial value was an item somewhere in the list and the user pressed down arrow, it actually selected the first item and ignored the position of the initial value (but that was fixed by this fix)